### DISCUSSION TITLE
NWS alert identifiers structure appears to have changed meaningfully recently

### DISCUSSION THREAD

**Discussion by amvtek** [weather-gov/api#218] [Q&A]
**Describe the bug**
We have an etl that loads NWS alerts from the `https://api.weather.gov/alerts` endpoint.
We have observed failure in the recent days (after `2021-02-24`) which are related to changes in alert identifier structure.

Before `2021-02-24` alert identifiers were as described in your api openapi schema (AlertId component):
* eg `NWS-IDP-PROD-4690566-3817270`

Now it looks a different structure is used
* eg `urn:oid:2.49.0.1.840.0.d9e2ee087e47de883a903df33ed0ff603f3ace1a.001.1`

This new structure does not match current AlertId component in openapi schema.

**To Reproduce**
Pull alerts using the ``https://api.weather.gov/alerts` endpoint in geo json format and checks the `id` field

**Expected behavior**
Identifier structure shall be documented and a maximum size should be set.

**Comment by StephenClouse** [comment]
This was an announced service change:

https://www.weather.gov/media/notification/pdf2/scn21-01cap_handler_v2aab.pdf

I have not uploaded the updated OpenAPI schema since adjusting for this change. I will do this later today.
